test/benchmarks/randomized_benchmarking.py

```diff
@@ -62,7 +62,7 @@ def build_rb_circuit(nseeds=1, length_vector=None,
 
 class RandomizedBenchmarkingBenchmark:
     # parameters for RB (1&2 qubits):
-    params = ([[[0]], [[0, 1]], [[0, 2], [1]]],)
+    params = ([[0, 3], [2], [1]])
```

Comment:
This is a step in the right direction, but I think will change the coverage somewhat (the original example had one single-qubit RB, one two-qubit RB, and one simultaneous RB example). 

```suggestion
    params = ([
        [[0]],  # Single qubit RB
        [[0, 1]],  # Two qubit RB
        [[0, 1], [2]]  # Simultaneous RB
    ],)
```


The benchmarks will be run with the `coupling_map` defined on L84, so qubits 0 and 1 are adjacent, but 0 and 2 (previously used for the simultaneous RB example) are not.
```
from qiskit.transpiler import CouplingMap
CouplingMap(
    [[1, 0], [1, 2], [2, 3], [4, 3], [4, 10], [5, 4],
     [5, 6], [5, 9], [6, 8], [7, 8], [9, 8], [9, 10],
     [11, 3], [11, 10], [11, 12], [12, 2], [13, 1],
     [13, 12]]
).draw()

        0 ← 1 →  2 →  3 ←  4 ← 5 → 6
            ↑    ↑    ↑    ↓   ↓   ↓
           13 → 12 ← 11 → 10 ← 9 → 8 ← 7

```

Reply:
Thank you for guiding @kdk. I have made the suggested changes, Please review